test/systemtest/earlgrey/test_fpga_nexysvideo.py

```diff
@@ -0,0 +1,126 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import os
+import re
+import time
+from pathlib import Path
+
+import pytest
+
+from .. import config, utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="module")
+def localconf_nexysvideo(localconf):
+    assert 'boards' in localconf
+    assert 'nexysvideo' in localconf['boards']
```

Comment:
I'm generally a fan of making an explicit "MyConfig" class and parsing the YAML to it, rather than passing around dictionaries. I'm not sure whether it's really worth the effort here, though. No strong feelings.